builtin/mainmenu/dlg_clients_list.lua

```diff
@@ -0,0 +1,59 @@
+--Luanti
+--Copyright (C) 2024 siliconsniffer
+--
+--This program is free software; you can redistribute it and/or modify
+--it under the terms of the GNU Lesser General Public License as published by
+--the Free Software Foundation; either version 2.1 of the License, or
+--(at your option) any later version.
+--
+--This program is distributed in the hope that it will be useful,
+--but WITHOUT ANY WARRANTY; without even the implied warranty of
+--MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--GNU Lesser General Public License for more details.
+--
+--You should have received a copy of the GNU Lesser General Public License along
+--with this program; if not, write to the Free Software Foundation, Inc.,
+--51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+--------------------------------------------------------------------------------
+
+local function clients_list_formspec(dialogdata)
+	local clients_list = dialogdata.server.clients_list
+	local servername   = dialogdata.server.name
+
+	local function fmt_formspec_list(clients_list)
+		local escaped = {}
+		for i, str in ipairs(clients_list) do
+			escaped[i] = core.formspec_escape(str)
+		end
+		return table.concat(escaped, ",")
+	end
+
+	return
+		"size[6,8]" ..
```

Comment:
Oh no, it seems you forgot to set a `formspec_version`, so you created the layout with the legacy formspec coordinate system! That's quite difficult and unpredictable compared to the current one...

Since the legacy formspec coordinate system has been legacy for a long time now (since 2019), adding a new formspec based on it doesn't make sense.

Reply:
That ... explains a lot. Fill add this, thanks for the hint.